source/content.ts

```diff
@@ -15,9 +18,11 @@ function getFieldValue(element: Element): string {
     return element.textContent;
 }
 
+
 function createLink(name: string, value: string): Element {
     const link = document.createElement("a");
-    const URL = `/app/discover#/?_a=(query:(language:kuery,query:'${name}:"${value}"'))`;
+    const _a = `_a=(query:(language:kuery,query:'${name}:"${value}"'))`;
+    const URL = `/app/discover#/?${KIBANA_G_TIME}&${_a}`;
```

Comment:
Seems like KIBANA_G_TIME can be empty, and in that way probably better to join conditionally like this:
```js
>> ["param1", "param2", "", null].filter(t => !!t).join("&")
"param1&param2"
```

Reply:
I made corrections.